ios/Podfile.lock

```diff
@@ -240,13 +240,13 @@ PODS:
   - React-jsinspector (0.63.4)
   - react-native-biometrics (2.1.4):
     - React
-  - react-native-camera (3.40.0):
```

Comment:
@diegoperini do we need this file changed? can we have it exactly as it was originally?

Reply:
We need it change at least for our package.